src/main/java/Deadline.java

```diff
@@ -0,0 +1,31 @@
+import java.time.LocalDate;
```

Comment:
You may want to package your classes so that contributors are better able to understand the code base!